packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -202,29 +203,31 @@ function Breadcrumbs<T>(props: SpectrumBreadcrumbsProps<T>, ref: DOMRef) {
   });
 
   return (
-    <nav
-      {...styleProps}
-      {...navProps}
-      ref={domRef}>
-      <ul
-        ref={listRef}
-        className={
-          classNames(
-            styles,
-            'spectrum-Breadcrumbs',
-            {
-              'spectrum-Breadcrumbs--small': size === 'S',
-              'spectrum-Breadcrumbs--medium': size === 'M',
-              'spectrum-Breadcrumbs--multiline': isMultiline,
-              'spectrum-Breadcrumbs--showRoot': showRoot,
-              'is-disabled': isDisabled
-            },
-            styleProps.className
-          )
-        }>
-        {breadcrumbItems}
-      </ul>
-    </nav>
+    <FocusScope>
```

Comment:
is this necessary for breadcrumbs? you can't really launch a dialog from there...

Reply:
Perhaps the intention was to make each of these components send focus back to the collection when the focused element is removed from the DOM (albeit these FocusScopes don't have `restoreFocus` provided to them...)